document_page_procedure/readme/CONFIGURE.md

```diff
@@ -0,0 +1,6 @@
+To be able configure your template of procedures, you need to install
+mgmtsystem_manual:
+
+- go to Management Systems \> Configuration \> Categories
+- select the Procedure category
+- customize the template
```

Comment:
```suggestion
To configure your template of procedures:

- Go to Management Systems \> Configuration \> Categories
- Select the Procedure category
- Customize the content
```

Reply:
If the mgmtsystem_manual module is not installed, we need to go to Knowledge > Pages > Categories instead.